pyairtable/orm/model.py

```diff
@@ -170,6 +172,29 @@ def _validate_class(cls) -> None:
                 )
             )
 
+    def modify_kwargs(func):
+        # Modifies kwargs passed to Model.all and Model.first
+        # If Meta.use_field_ids, kwargs['return_fields_by_field_id']=True.
+
+        @wraps(func)
+        def wrapper(cls, *args, **kwargs):
+            kwargs = cls._kwargs_all_first(kwargs)
+            return func(cls, *args, **kwargs)
+
+        return wrapper
+
+    @classmethod
+    def _kwargs_all_first(cls, request_args: dict):
+        # Called by modify_kwargs to modify kwargs passed to `all()` & `first()`.
+        disallowed_args = (
+            "cell_format",
+            "return_fields_by_field_id",
+        )
+        args_union = {k: v for k, v in request_args.items() if k not in disallowed_args}
+        if cls._get_meta("use_field_ids"):
+            args_union["return_fields_by_field_id"] = True
+        return args_union
```

Comment:
If we follow the pattern above, I think we could probably simplify this down to something like:

```
return {
    "cell_format": "json",
    "user_locale": None,
    "time_zone": None,
    "return_fields_by_field_id": cls._get_meta("use_field_ids"),
}
```

I also wonder if this would be a good place to put `kwargs.setdefault("fields", ...)` so that we don't retrieve more data from the API than our ORM actually requires. Right now the ORM will retrieve values for all fields on a table, regardless of whether they're mapped to a field descriptor.